Update Protocols FAQ (#6158)
segmentio/segment-docs: src/protocols/faq.md

Blocked events are blocked from sending to all Segment Destinations, including warehouses and streaming Destinations. When an Event is blocked using a Tracking Plan, it does not count towards your MTU limit. They will, however, count toward your MTU limit if you enable [blocked event forwarding](/docs/protocols/enforce/forward-blocked-events/) in your Source settings.

### Why am I seeing unplanned properties/traits in the payload when violations are triggered, despite using schema controls to omit them?

> Segment's [Schema Controls](docs/connections/sources/schema/destination-data-control/) provide three options to omit properties/traits. Select the one that aligns with your requirements:

Reviewer: ```suggestion
If you're seeing unplanned properties/traits in your payload despite using Schema Controls, you might want to select a new degree of blocking controls. 

Segment's [Schema Controls](docs/connections/sources/schema/destination-data-control/) provide three options to omit properties/traits. Select the one that aligns with your requirements:
```

> [Segment's Schema Controls]( https://segment.com/docs/connections/sources/schema/destination-data-control/) provide three options to omit properties/traits. Ensure you've chosen the one that aligns with your requirements:

Reviewer: ```suggestion
Segment's [Schema Controls](docs/connections/sources/schema/destination-data-control/) provide three options to omit properties/traits. Select the one that aligns with your requirements:
```

> 1. **Standard Schema Controls/"Unplanned Properties/Traits"**: Selecting these options means Segment checks the names of incoming properties/traits against your Tracking Plan.

Reviewer: ```suggestion
1. **Standard Schema Controls/"Unplanned Properties/Traits"**: Segment checks the names of incoming properties/traits against your Tracking Plan.
```

> 2. **Standard Schema Controls/"JSON Schema Violations"**: With this option, Segment not only checks the names but also evaluates the values of properties/traits. This is useful if you've specified a pattern or a list of acceptable values in the [JSON schema](https://segment.com/docs/protocols/tracking-plan/create/#edit-underlying-json-schema) for each Track event listed in the Tracking Plan.

Reviewer: ```suggestion
2. **Standard Schema Controls/"JSON Schema Violations"**: Segment checks the names and evaluates the values of properties/traits. This is useful if you've specified a pattern or a list of acceptable values in the [JSON schema](/docs/protocols/tracking-plan/create/#edit-underlying-json-schema) for each Track event listed in the Tracking Plan.
```

> 3. **Advanced Blocking Controls/"Common JSON Schema Violations"**: This option allows Segment to evaluate incoming events thoroughly, including event names, context field names and values, and the names and values of properties/traits, against the [Common JSON schema](https://segment.com/docs/protocols/tracking-plan/create/#common-json-schema) in your Tracking Plan.

Reviewer: ```suggestion
3. **Advanced Blocking Controls/"Common JSON Schema Violations"**: Segment evaluates incoming events thoroughly, including event names, context field names and values, and the names and values of properties/traits, against the [Common JSON schema](/docs/protocols/tracking-plan/create/#common-json-schema) in your Tracking Plan.
```
